packages/itwin/tree-widget/src/test/trees/models-tree/ModelsTreeFiltering.test.ts

```diff
@@ -1271,29 +1341,119 @@ describe("Models tree", () => {
                             children: [
                               NodeValidators.createForClassGroupingNode({
                                 label: "Physical Object",
-                                autoExpand: true,
-                                children: [
-                                  NodeValidators.createForInstanceNode({
-                                    instanceKeys: [x.physicalElement1],
-                                    label: /^element 1/,
-                                  }),
-                                  NodeValidators.createForInstanceNode({
-                                    instanceKeys: [x.physicalElement2],
-                                    label: /^element 2/,
-                                  }),
-                                ],
+                                autoExpand: false,
+                                groupedInstanceKeys: x.physicalElementGroup.groupedElements,
                               }),
                               NodeValidators.createForClassGroupingNode({
                                 label: "Test Physical Object",
+                                autoExpand: false,
+                                groupedInstanceKeys: x.testElementGroup.groupedElements,
+                              }),
+                            ],
+                          }),
+                        ],
+                      }),
+                    ],
+                  }),
+                ],
+              }),
+            ],
+          }),
+        ],
+      ),
+      TreeFilteringTestCaseDefinition.create(
+        "deeply nested grouping nodes",
```

Comment:
We're not filtering grouping nodes, so let's make the test name more specific. Maybe "grouped elements at different hierarchy levels"? Please review names of the other new tests as well.

Reply:
There were already 4 test cases added by @saskliutas that have names like that, e.g. "grouping element node under category". I just followed the example. Should I rename all of those as well?

Reply:
I don't agree the names are "like that", but I agree that their names could use an improvement.

Saulius' test names mention a "grouping element" it's unclear what that really is. If I didn't see the test source, I'd probably think it's the source of `bis.ElementGroupsMembers` relationship. I think what Saulius meant was "grouped element" - can you please make the change?

Your test names mention "grouping node", which can't be a filter target. I suggest these renames:
- "deeply nested grouping nodes" -> "elements' hierarchy, when auto-expanding their grouping nodes",
- "grouping node and element" -> "element, when auto-expanding up to element and its grouping node",
- "grouping nodes under different categories" -> "elements under different categories, when auto-expanding up to their grouping nodes"

Reply:
Renamed the tests